examples/benchmarks/resnet_imagenet/main.py

```diff
@@ -7,6 +7,11 @@
 import sys
 from typing import Dict
 
+# Add folder root to path to allow us to use relative imports regardless of what directory the script is run from
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
+
+import data as data_module
+import model as model_module
```

Comment:
why this instead of importing the necessary functions directly

Reply:
idk, seemed safer. not really attached to anything in particular, but would also rather not change it at this point